recipes/papaa/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "0.1.5" %}
+
+package:
+  name: papaa
+  version: '{{version}}'
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/nvk747/papaa/archive/papaa-{{ version }}.tar.gz
+  sha256: c117e9296cf43ff10490449280d61b4da60963e23d146a50fcc9fbc7c636c430
+
+requirements:
+  run:
+    - python==3.6.5
```

Comment:
Does it really need exactly python 3.6.5? So 3.7 through 3.9 don't work? Likewise, does numpy >=1.14.3 not work for some reason?